packages/sections/src/variant/VariantEffectPredictor/Body.tsx

```diff
@@ -13,23 +14,32 @@ function formatVariantConsequenceLabel(label) {
 
 const columns = [
   {
-    id: "transcriptId",
-    label: "Transcript",
-    tooltip: "Ensembl canonical transcript",
+    id: "target.approvedSymbol",
+    label: "Gene",
     comparator: (a, b) => a.transcriptIndex - b.transcriptIndex,
-    renderCell: ({ transcriptId, target }) => {
-      if (!transcriptId) return naLabel;
-      if (!target) return transcriptId;
-      return (
-        <Link
-          external
-          to={`https://www.ensembl.org/Homo_sapiens/Transcript/Summary?db=core;g=${
-               target.id};t=${transcriptId}`}
-        >
-          {transcriptId}
-        </Link>
-      )
-    }
+    renderCell: ({ target, transcriptId }) => (
```

Comment:
code flow of this logi can be much simpler here:
ex:
` renderCell: ({ target, transcriptId }) => {
  if(!target) return naLabel;
  let displayElement = <Link to=target.id}>{target.approvedSymbol}</Link>;
  if(transcriptId) {
    displayElement = <Tooltip> {displayElement}</Tooltip>
  }
  return displayElement;
} `

Reply:
Agreed - changes made.